ratatui-core/src/buffer/cell.rs

```diff
@@ -128,12 +147,9 @@ impl Cell {
         }
     }
 
-    /// Sets the cell to be skipped when copying (diffing) the buffer to the screen.
-    ///
-    /// This is helpful when it is necessary to prevent the buffer from overwriting a cell that is
-    /// covered by an image from some terminal graphics protocol (Sixel / iTerm / Kitty ...).
-    pub fn set_skip(&mut self, skip: bool) -> &mut Self {
-        self.skip = skip;
+    /// Sets cell diffing options.
+    pub fn set_diff_option(&mut self, diff_option: CellDiffOption) -> &mut Self {
+        self.diff_option = diff_option;
```

Comment:
One thing we tend to do to make it easier for upgrading between versions is to make changes in a backwards compatible manner. Could this be done by deprecating `set_skip()` and having it call `set_diff_option` with the right value?

There's only a small amount of usages of the existing method, so this might not be a huge deal. https://github.com/search?q=ratatui+set_skip+AND+%28NOT+org%3Aratatui%29+AND+%28NOT+is%3Afork%29&type=code